src/dynamics/time_integration.jl

```diff
@@ -106,13 +106,12 @@ be a divisor such that an integer number of time steps matches exactly with the
 time step."""
 function initialize!(L::Leapfrog, model::AbstractModel)
     (; output_dt) = model.output
+    (; radius) = model.planet
 
-    if L.adjust_with_output
```

Comment:
Is the check not needed anymore?

Reply:
it's incorrectly placed, the actual check should happen in `get_dt_millisec` which it does but even with `adjust_with_output = false` the time steps should be recomputed here!